stories/landfill-ej.stories.mdx

```diff
@@ -0,0 +1,202 @@
+---
+id: 'landfill-ej'
+name: Disproportionate Environmental Burdens - A Study of Landfills
+media:
+  src: ::file ./landfill.jpg
+  alt: Garbage pile in landfill.
+  author:
+    name: vchal
+    url: https://www.istockphoto.com/photos/landfill
+pubDate: 2024-04-05
```

Comment:
might be better to change this to more recent date?

Reply:
We can change the pub date to today, or whenever it's actually pushed to prod!